services/orchestrator/src/main/java/org/eclipse/osc/orchestrator/MemoryOrchestratorStorageImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.osc.orchestrator;
+
+import org.apache.karaf.minho.boot.spi.Service;
+
+import java.util.Set;
+import java.util.concurrent.ConcurrentSkipListSet;
+
+public class MemoryOrchestratorStorageImpl implements OrchestratorStorage, Service {
```

Comment:
Should we keep this memory storage way? As discussed last time, we are supposed to have a local file storage impl at least.

Reply:
Yes. Good point. Let me update the PR with local file system storage and provide another distributed one (powered by Hazelcast for instance) as example.